transaction/core/src/ring_ct/rct_bulletproofs.rs

```diff
@@ -95,6 +99,41 @@ pub struct SignedInputRing {
     pub signed_digest: Option<[u8; 32]>,
 }
 
+/// Signature data that is required for any signer to produce signature
+/// and construct transaction
+#[derive(Clone, Debug)]
+pub struct ViewOnlySigningData {
```

Comment:
I think `ViewOnly` is not helpful here - what are we trying to communicate here?
Is this structure on its own sufficient to produce a signature (I don't think so since it lacks the individual ring signatures)? And if not, as I suspect is the case, how do we better explain what it does contain?

Reply:
I'm trying to communicate that this is not all of the information required to produce a signature, but the required materials that are attainable with the view private key. I'm fine with name changes, if there are thoughts on that